app/queries/display_ads/filtered_ads_query.rb

```diff
@@ -56,6 +59,12 @@ def authenticated_ads(display_auth_audience)
       @filtered_display_ads.where(display_to: display_auth_audience)
     end
 
+    def community_or_in_house_ads
+      # community type ads matching author OR any/all in-house ads
+      # Whats considered an "ads matching author"?
+      @filtered_display_ads.where(type_of: %w[community in_house])
```

Comment:
As I understand it, what I think what we want here is **community type ads matching author OR any/all in-house ads**. Keeping in mind that Rails will only _sometimes_  translate the [`enum` fields](https://github.com/forem/forem/blob/main/app/models/display_ad.rb#L19) for us — these look like strings in Ruby, but they are [integers in the database](https://github.com/forem/forem/blob/main/db/migrate/20221110130111_add_type_to_display_ads.rb#L3) — basically, it will if we're using Arel-style queries, but it won't if the query becomes more SQL-ified.

So, something like this:
 
```suggestion
      @filtered_display_ads.where(
        "(type_of = :in_house) OR 
         (type_of = :community AND organization_id = :organization_id)", 
          DisplayAd.type_ofs.merge({organization_id: organization_id})
      )
```

We'd still need to have passed in an `organization_id` in the initializer for this to work, so callers would need to be updated, as well.